tests/QueryBuilderTest.php

```diff
@@ -38,11 +38,31 @@ public function setUp(): void
      */
     public function testConstruct()
     {
-        $builder = new QueryBuilder('Object');
+        $builder = new QueryBuilder('Object', 'ObjectAlias');
```

Comment:
I would rather if we move this to a separate test to keep the `testConstruct` for the basic base case.

Reply:
No problem. That is done for you now. Sorry about the delay.